sceptre/nlpsandbox/config/common/nlpsandboxvpc.yaml

```diff
@@ -0,0 +1,12 @@
+template_path: remote/vpc-mini.yaml
+stack_name: nlpsandboxvpc
+stack_tags:
+  Department: "CompOnc"
+  Project: "NLP Sandbox"
+  OwnerEmail: "thomas.yu@sagebase.org"
+parameters:
+  VpcSubnetPrefix: "10.255.21"
```

Comment:
Confirmed, recorded here; https://sagebionetworks.jira.com/wiki/spaces/IT/pages/352976898/Sage+VPN